src/Model/Entity/Sentence.php

```diff
@@ -86,4 +95,24 @@ protected function _getOldFormat()
 
         return $result;
     }
+
+    protected function _getLangName()
+    {
+        return LanguagesLib::getLanguageName($this->lang);
+    }
```

Comment:
I have created `src/Model/Entity/LanguageNameTrait.php` for that. See 56497bfb748fb88eed7308595fb64eba27790d1f.